src/nrniv/neuronapi.h

```diff
@@ -132,6 +132,14 @@ Symlist* nrn_global_symbol_table(void);
 Symlist* nrn_top_level_symbol_table(void);
 int nrn_symbol_array_length(const Symbol* sym);
 void nrn_register_function(void (*proc)(), const char* func_name, int type);
+/****************************************
+ * Parameter-reading functions
+ ****************************************/
+Object** nrn_objgetarg(int arg);
+char* nrn_gargstr(int arg);
+double* nrn_getarg(int arg);
+IvocVect* nrn_vector_arg(int arg);
```

Comment:
Let's drop `nrn_vector_arg` because because Vectors aren't things that we provide direct access to. This isn't a major limitation because we can get objargs and the API does have special access for Object* that are Vector